### PR TITLE
Playlist management

### PR DESCRIPTION
**Pull request by sameep-s**
Added Features:
Create playlists,
delete playlists
remove video from playlist,
add video to playlist

### CODE REVIEW

**Review by tewarimohit955 — APPROVED** [review]
good to merge..

File: src/components/SuggestionVideos/SuggestionVideo.jsx [hunk 1]
```diff
@@ -1,33 +1,36 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import "./suggestionVideo.css";
 
-const SuggestionVideo = () => {
+const SuggestionVideo = (video) => {
+
```
Comment (arjunbharti774) [inline src/components/SuggestionVideos/SuggestionVideo.jsx:6]: avoid extra spacing in the code

**Review by arjunbharti774 — APPROVED** [review]
Really liked the folder structure and well written contexts.
Suggestion - avoid extra spacing in the code. It makes the code more clean
Everything else looks good to me!

File: src/components/SuggestionVideos/SuggestionVideo.jsx (hunk 1 above)
Comment (sameep-s) [reply src/components/SuggestionVideos/SuggestionVideo.jsx:6]: sure i'll take care of it

**Pull request merged**